functions-and-scope.js

```diff
@@ -78,6 +141,24 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+/*-------------------CODE------------------*/
+const grades = [8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+
+function highestgrade(cijfers) {
+    let bestgrade = cijfers[0];
+    for (let i = 0; i < cijfers.length; i++) {
+        if (bestgrade <= cijfers[i+1]) {
```

Comment:
De constructie met i+1 is niet echt logisch. Je zou namelijk per iteratie de variabele bestgrade kunnen vergelijken met de huidige waarde in de array. Mocht de huidige waarde groter zijn dan bestgrade, dan plaats je die waarde in bestgrade. Die wordt per iteratie herhaald waardoor uiteindelijk altijd het hoogste cijfer terecht komt in jouw variable bestgrade.